src/site/topic13.rst

```diff
@@ -2,6 +2,31 @@
 Topic #13 --- Inheritance
 *************************
 
+* Inheritance is a way to derive a new class from an existing one
+* This allows us to *inherit* functionality from the original class
+
+* Consider the hierarchies below
+* Notice how the top element is the most general of all things included
+* As you move down the three, the things get more and more specific
+* But as you move down, the things are still a type of the element at the top
+    * A square *is a* rectangle, which *is a* 2d shape, which *is a* shape
```

Comment:
_is a_ is problematic. Maybe _inherits from_ or _is a subtype of_?

And, naturally, a square is **not** a rectangle. Best not to think of the real world objects here, rather the classes in their representation space.